app/src/lib/editors/linux.ts

```diff
@@ -212,6 +212,10 @@ const editors: ILinuxExternalEditor[] = [
       '/usr/bin/zed',
     ],
   },
+  {
+    name: 'Helix',
+    paths: ['/usr/bin/hx'],
```

Comment:
i suggest adding `/usr/bin/env hx` also as i did this job in my PR.

Reply:
Done @s0me1newithhand7s